src/benchmarks/micro/corefx/System.Text.Json/Serializer/ReadPreservedReferences.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !NETFRAMEWORK && !NETCOREAPP2_1 && !NETCOREAPP2_2 && !NETCOREAPP3_0 && !NETCOREAPP3_1
```

Comment:
I think that as for now we can simplify it to `#if NETCOREAPP5_0` and later when we have more 5.0-only files extend the project file with a new rule for 5.0 

https://github.com/dotnet/performance/blob/ca80d8e2886b583d0a69635740b188248d3d6fdd/src/benchmarks/micro/MicroBenchmarks.csproj#L73

```suggestion
#if NETCOREAPP5_0
```